src/main/scala/cromwell/engine/WorkflowDescriptor.scala

```diff
@@ -55,10 +56,18 @@ case class WorkflowDescriptor(id: WorkflowId,
   private val props = sys.props
   private val relativeWorkflowRootPath = s"$name/$id"
   private val log = WorkflowLogger("WorkflowDescriptor", this)
+  val workflowLogDir = workflowOptions.get("workflow_log_dir") recover { case e: IllegalArgumentException =>
```

Comment:
Yeah, I spent a bit looking at this... these three are independent and the errors are never really "thrown", only logged.

Each value here is actually of type `Try[String]`.  Their `throw`s are actually [caught](https://github.com/scala/scala/blob/v2.11.7/src/library/scala/util/Try.scala#L214-L220) by `recover` and go right back to being the same `Failure`. Still, during each `WorkflowDescriptor` construction, they each independently log if anything is wrong. After construction, at the time of access, the `Failure` is quietly ignored with a `Failure.getOrElse(Future.successful(()))`.

A NEL validating the `WorkflowDescriptor` may be better, but I had to draw a line somewhere when choosing how much to patch in this PR.

Will leave it up to reviewers to decide if this copypasta behavior should be updated as well in this PR?